easybuild/easyconfigs/c/CP2K/CP2K-2022.1-foss-2022a.eb

```diff
@@ -0,0 +1,40 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'CP2K'
+version = '2022.1'
+
+homepage = 'https://www.cp2k.org/'
+description = """CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
+ simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different
+ methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and
+ classical pair and many-body potentials. """
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+source_urls = ['https://github.com/cp2k/cp2k/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['2c34f1a7972973c62d471cd35856f444f11ab22f2ff930f6ead20f3454fd228b']
+
+#                https://github.com/cp2k/cp2k/releases/download/v2022.1/cp2k-2022.1.tar.bz2
+dependencies = [
+    ('Libint', '2.6.0', '-lmax-6-cp2k'),
```

Comment:
@alinelena Please update to 2.7.2 (see #16073), and remove the Libint easyconfig file from this PR

Reply:
fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16007/commits/8d95a7d96fd1ed37c1752c5830ee327c1a9ed95d